src/alliteratedActors.js

```diff
@@ -22,7 +22,17 @@
  * @return {string}
  */
 function alliteratedActors(names) {
-  // write code here
+  let namesSame = [];
```

Comment:
this name is a bit misleading. I'd understand as 'name are equal'. Perhaps 'alliteratedNames'?